letsencrypt/client/init.sls

```diff
@@ -13,6 +13,7 @@ certbot_packages:
     - names: {{ client.source.pkgs }}
     - watch_in:
       - cmd: certbot_installed
+    - fromrepo: jessie-backports
```

Comment:
yes, I see. I wont check salt-formula-linux, because I don't have any time for that. Would you be OK for a pillar value? I installed a backports repos with another state, but how to tell `pkg.installed` to use it without passing `- fromrepo`?